src/main.rs

```diff
@@ -202,19 +202,21 @@ struct Args {
     /// If provided, allows to set a custom Max Execution time for the Host in ms.
     #[clap(long = "max-execution-time-ms", default_value = "600000", env = "WASMCLOUD_MAX_EXECUTION_TIME_MS", value_parser = parse_duration_millis)]
     max_execution_time: Duration,
-    /// The maximum amount of memory bytes that a component can allocate (default 256 MiB)
-    #[clap(long = "max-linear-memory-bytes", default_value_t = 256 * 1024 * 1024, env = "WASMCLOUD_MAX_LINEAR_MEMORY")]
-    max_linear_memory: u64,
-    /// The maximum byte size of a component binary that can be loaded (default 50 MiB)
-    #[clap(long = "max-component-size-bytes", default_value_t = 50 * 1024 * 1024, env = "WASMCLOUD_MAX_COMPONENT_SIZE")]
-    max_component_size: u64,
-    /// The maximum number of components that can be run simultaneously
+    /// This option is deprecated, do not use
```

Comment:
Re above comment, this could be made backwards compatible.

For this PR, it would be great if this was more descriptive unless we truly just don't pipe this through. A recommendation to use the other env var would be good